src/applications/facility-locator/config.js

```diff
@@ -127,6 +126,7 @@ export const healthServices = {
   MentalHealthCare: 'Mental health care',
   DentalServices: 'Dental services',
   UrgentCare: 'Urgent care',
+  Covid19Vaccine: 'COVID-19 vaccines',
```

Comment:
This is a temporary parameter, we will have to confirm with lighthouse / backend support

Reply:
The key, `Covid19Vaccine`, should probably be a shared constant.